data/pqtl/02.analyze-GpBPpG.ipynb

```diff
@@ -165,20 +159,20 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0 took 5.1 seconds\n",
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-1 took 6.6 seconds\n",
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-2 took 6.5 seconds\n",
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-3 took 6.6 seconds\n",
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-4 took 6.6 seconds\n",
-      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-5 took 6.5 seconds\n"
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0 took 1.9 seconds\n",
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-001 took 3.1 seconds\n",
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-002 took 3.1 seconds\n",
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-003 took 3.1 seconds\n",
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-004 took 3.1 seconds\n",
+      "Computing DWPC matrix for the GpBPpG metapath in hetionet-v1.0-perm-005 took 3.2 seconds\n"
      ]
     }
    ],
    "source": [
     "DWPCs = collections.OrderedDict()\n",
-    "for name, graph in hetnets.items():\n",
-    "    metapath = graph.metagraph.metapath_from_abbrev('GpBPpG')\n",
-    "    row_ids, col_ids, dwpc_matrix, seconds = hetmech.degree_weight.dwpc(graph, metapath, damping=0.4, dense_threshold=0.6)\n",
+    "for name, permat_ in hetmats.items():\n",
```

Comment:
Why the trailing underscore?

Reply:
I meant `hetmat_` instead of `permat_`. I used a single trailing underscore since I didn't want to overwrite the `hetmat` variable which contains the original unpermuted hetnet.